.gitignore

```diff
@@ -13,3 +13,7 @@
 
 # General
 .DS_Store
+
+# Local dev
+/config/test.env
+owner_test.go
```

Comment:
Why add this? Seems like a local conf?

Reply:
It's in the ignore so my local env file does not ever get checked in. The last thing I want is to check in any sensitive env params that I'm using my development by mistake. Plus using env config is useful for test.env, stage.env, or production.env.